fix: distinguish header and logo colors, added proper logo styles

Tanmayshi opened the pull request:
fix: distinguish header and logo colors, added proper logo styles
Improvement: Updated the logo colors to align with the proper color scheme for better visual clarity.
Additional Change: Enhanced the UI by improving the contrast between header and logo elements, making it more user-friendly.
Related issue(s)
Fixes #484

TenzDelek commented on components/Sponsors/sponsors.js:
```diff
@@ -17,7 +17,7 @@ function Sponsors({eventSponsors,financialSponsor}) {	 //eventSponsors,financial
 					</Paragraph>
 				</div>
-				<Heading typeStyle='heading-md' className='text-white mt-10 sm:text-2xl sm:mt-5'>Event and Host Sponsor</Heading>
+				<Heading typeStyle='heading-md' className='text-gray-800 mt-10 sm:text-2xl sm:mt-5'>Event and Host Sponsor</Heading>
```
a lighter shade is appreciated. (also try with stale)

TenzDelek commented on pages/index.js:
```diff
@@ -186,7 +186,7 @@ export default function Home() {
 			</div>
 			<div id='sponsors' className='mt-20'>
-				<Sponsors eventSponsors={[{image:'/img/apidays.png',websiteUrl:"https://www.apidays.global/"}]} financialSponsor={[{image:"/img/graviteeio.svg" , websiteUrl: "https://www.gravitee.io/",className:"w-[250px] h-[50px]" } , {image:"/img/postman.png" ,websiteUrl:"https://www.postman.com/",className:"w-[240px] h-[70px]"}]}  />
+				<Sponsors eventSponsors={[{image:'/img/apidays_2.webp',websiteUrl:"https://www.apidays.global/"}]} financialSponsor={[{image:"/img/graviteeIo_1.png" , websiteUrl: "https://www.gravitee.io/",className:"w-[250px] h-[50px]" } , {image:"/img/postman_1.png" ,websiteUrl:"https://www.postman.com/",className:"w-[240px] h-[70px]"}]}  />
```
I believe there is no need to change the logo, the logo are taken from the official branding i believe

AceTheCreator reviewed (changes requested):
Kindly fix your PR title as suggested by the bot :)

Tanmayshi commented:
"Hi @AceTheCreator,
I have updated the PR title as per the Conventional Commit guidelines (fix: distinguish header and logo colors, added proper logo styles). Please review the changes and let me know if there's anything else to address. 😊"

AceTheCreator commented:
> "Hi @AceTheCreator, I have updated the PR title as per the Conventional Commit guidelines (fix: distinguish header and logo colors, added proper logo styles). Please review the changes and let me know if there's anything else to address. 😊"

I think you should apply suggested changes from @TenzDelek 🤔

TenzDelek commented:
I think the contributor don't want to work on this issue looking at the deleted branch. Therefore I will be taking the issue from here
